canonical-websites/www.canonical.com
# Upgrade /products to v1

## Done

Upgrade /products to v1

## QA

- Pull code
- Check markup at `templates/products/index.html`
- Run `./run serve --watch`
- Check page locally at http://0.0.0.0:8002/products

Fixes https://www.github.com/ubuntudesign/vanilla-squad/issues/26

## Demo

http://www.canonical.com-homepage-polish.demo.haus/products
